src/pages/settings/Settings.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="flex flex-col space-y-6 md:space-y-4">
+    <Navigation />
```

Comment:
![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/409fa252-c8f1-4573-b2ac-a604c1220978)

Breadcrumbs are handed [here](https://github.com/epicmaxco/vuestic-admin/pull/986). My bad I didn't mention that in the issue 😓.

So you can safely ignore these in this PR and just focus on page itself.